src/main/java/io/xstefank/wildlfy/bot/format/checks/CommitsQuantityCheck.java

```diff
@@ -17,7 +17,7 @@ public class CommitsQuantityCheck implements Check {
     private String message;
 
     public CommitsQuantityCheck(CommitsQuantity commitsQuantity) {
-        if (commitsQuantity == null || commitsQuantity.quantity == null) {
```

Comment:
I don't understand this. So this wasn't needed? Or if user configures check root but not the actual required configuration under it then it shouldn't fail?

Reply:
This check is not reachable. Because I made a check for this here in **PullRequestFormatVerifier**:
`if (wildflyConfigFile.wildfly.format.commitsQuantity != null) {
            checks.add(new CommitsQuantityCheck(wildflyConfigFile.wildfly.format.commitsQuantity));
        }`